src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__rating-stars {
```

Comment:
It looks like card element and if you create it to use BEM mix methodology then you need to keep this class in card file but make it separate from stars block